templates/download.html

```diff
@@ -82,11 +84,12 @@ <h2>Ubuntu Server</h2>
     <div class="col-6 p-card">
       <h3 class="p-card__title">Ubuntu Server {{ releases.lts.full_version }} LTS</h3>
       <div class="p-card__content">
-        <p>Ubuntu ServerのLTS版には、OpenStackのQueensリリースが含まれ、2023年4月までのサポートが保証されています。64ビット版のみの提供です。</p>
+        <p>Ubuntu ServerのLTS版には、OpenStackのQueensリリースが含まれ、{{ releases.lts.eol }}年4月までのサポートが保証されています。64ビット版のみの提供です。</p>
```

Comment:
Isn't openstack "U"?

Reply:
yes, it wasn't using the variable... fixed